gym_electric_motor/envs/gym_eesm/cont_cc_eesm_env.py

```diff
@@ -0,0 +1,165 @@
+
+    Initial State:
+        Zeros on all state variables.
+
+    Example:
+        >>> import gym_electric_motor as gem
+        >>> from gym_electric_motor.reference_generators import LaplaceProcessReferenceGenerator
+        >>>
+        >>> # Select a different ode_solver with default parameters by passing a keystring
+        >>> my_overridden_solver = 'scipy.solve_ivp'
+        >>>
+        >>> # Update the default arguments to the voltage supply by passing a parameter dict
+        >>> my_changed_voltage_supply_args = {'u_nominal': 400.0}
+        >>>
+        >>> # Replace the reference generator by passing a new instance
+        >>> my_new_ref_gen_instance = LaplaceProcessReferenceGenerator(
+        ...     reference_state='i_sq',
+        ...     sigma_range=(1e-3, 1e-2)
+        ... )
+        >>> env = gem.make(
+        ...     'Cont-CC-EESM-v0',
+        ...     voltage_supply=my_changed_voltage_supply_args,
+        ...     ode_solver=my_overridden_solver,
+        ...     reference_generator=my_new_ref_gen_instance
+        ... )
+        >>> done = True
+        >>> for _ in range(1000):
+        >>>     if done:
+        >>>         state, reference = env.reset()
+        >>>     env.render()
+        >>>     (state, reference), reward, done, _ = env.step(env.action_space.sample())
+    """
+    def __init__(self, supply=None, converter=None, motor=None, load=None, ode_solver=None,
+                 reward_function=None, reference_generator=None, visualization=None, state_filter=None, callbacks=(),
+                 constraints=(SquaredConstraint(('i_sq', 'i_sd')), LimitConstraint(('i_e',))), calc_jacobian=True, tau=1e-4):
+        """
+        Args:
+            supply(env-arg): Specification of the :py:class:`.VoltageSupply` for the environment
+            converter(env-arg): Specification of the :py:class:`.PowerElectronicConverter` for the environment
+            motor(env-arg): Specification of the :py:class:`.ElectricMotor` for the environment
+            load(env-arg): Specification of the :py:class:`.MechanicalLoad` for the environment
+            ode_solver(env-arg): Specification of the :py:class:`.OdeSolver` for the environment
+            reward_function(env-arg): Specification of the :py:class:`.RewardFunction` for the environment
+            reference_generator(env-arg): Specification of the :py:class:`.ReferenceGenerator` for the environment
+            visualization(env-arg): Specification of the :py:class:`.ElectricMotorVisualization` for the environment
+            constraints(iterable(str/Constraint)): All Constraints of the environment. \n
+                - str: A LimitConstraints for states (episode terminates, if the quantity exceeds the limit)
+                 can be directly specified by passing the state name here (e.g. 'i', 'omega') \n
+                - instance of Constraint: More complex constraints (e.g. the SquaredConstraint can be initialized and
```

Comment:
I will not comment this for each individual entry, but please try to always put a comma after "e.g.":
"e.g., the SquaredContraint ..."